src/main.go

```diff
@@ -1,21 +1,27 @@
 package main
 
-import  (
+import (
 	"flag"
+	"strings"
+
+	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/Merger"
 	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/SpecReader"
 	html "github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/HTML"
 )
 
 func main() {
 	file := flag.String("file", "", "This takes a path to a csv file/string")
+	merge := flag.String("merge", "", "This takes paths to files that should be merged (space separated)")
 	flag.Parse()
 
 	var specs SpecReader.Blips
 	// testing csv reader
 	if *file != "" {
 		specs = SpecReader.ReadCsvSpec(*file)
-	} else {
-		panic("No file was given (oh no)")
```

Comment:
Why do we remove the else stm? O:

Might it be bc it propagates the lack of file-input somewhere else, so it still spits out a relevant message? o:

Reply:
I'm assuming it's to make it not give an error when calling the merger but this seems like a very hacky solution to do this. This would however be solved by moving to cobra so I would say lets just wait till that happens before spending too much time with this specific version

Reply:
If people want a slightly cleaner but still very hacky solution feel free to look at the feat_git_fetcher branch